settings.py

```diff
@@ -2,11 +2,7 @@
 
 DEBUG = True
```

Comment:
Seems like we do not use the env var?

Reply:
In my opinion we should read an env var here, don't you agree? Otherwise its not possible to deactivate the debug mode for CAT and PROD. Not sure if default should be true or false.